code/modules/arcade/mob_hunt/game_server.dm

```diff
@@ -0,0 +1,164 @@
+var/global/datum/controller/process/mob_hunt/mob_hunt_server
```

Comment:
You mean exactly like the seed controller, which is likewise located with its relevant module files?